tests/integration/docs_generate_tests/test_docs_generate.py

```diff
@@ -516,6 +518,7 @@ def rendered_snapshot_config(self, **updates):
             'unique_key': 'id',
             'target_schema': None,
             'meta': {},
+            'unique_key': 'id'
```

Comment:
Is this a duplicate of line 518?

Reply:
Shoot! An artifact of debugging and trying to parse those darn diffs. Didn't realize it would pass duplicates. Thanks for noticing.